yunohost_project_organization.md

```diff
@@ -1,41 +1,41 @@
 # YunoHost project organisation
 
-## Document purpose
+## Purpose
 
-This document aims to allow contributors to feel legitimate in contributing to the YunoHost project together with collective feedback.
-The project is community-based and hasty decisions made by a restricted group of contributors can generate frustrations at a later stage.
-To address this issue, the proposed solution here is to ensure that decisions are taken collectively and that they are sufficiently thought out.
-An advisory council provides orientations for the evolution of the YunoHost project and special interest groups allow more efficient contribution in relation to each specific topic.
+Feel legitimized in contributing to the YunoHost project and part-take in the collective feedback.
+The project is community-based to avoid hasty decisions made by a restricted group of contributors generating frustrations at a later stage.
+Decisions are taken collectively, when sufficiently thought out.
+An advisory council orients the evolution of the YunoHost project, and special interest groups allow more efficient contribution in each specific area.
 
 ## Definition of YunoHost
 
 ### Purpose
 
-The goal of YunoHost is to make the installation and administration of a server accessible to the largest number of people, without prejudice to the quality and reliability of the software. 
+Installing and administrating a server should be accessible to the largest number of people, without prejudice to the quality and reliability of the software. 
```

Comment:
Hmokay but the initial sentence was clearly stating "The goal of Yunohost is ..." whereas the new sentence sounds more like a wish..?

Reply:
That part is redundant. In that vein, and as long as the aim is not obviously met, the rest can serve as a reminder of what it should be. I don't understand "without prejudice to the quality and reliability of the software." btw. To me it means the installation and admin part of anything on a server should be accessible, even if the software is bad and unreliable. Meaning YunoHost somehow facilitates bad external software, reading into that on a half-empty glass.